site/website/versioned_docs/version-4.1/operations/docker.md

```diff
@@ -0,0 +1,349 @@
+---
+title: Hazelcast Jet with Docker
+description: How to use Jet in a Docker environment
+id: version-4.1-docker
+original_id: docker
+---
+
```

Comment:
Notice the above comments for this document, too.

Reply:
Made same changes to both docs.